conferences/abstracts/OSF19_Demo_submission.md

```diff
@@ -0,0 +1,91 @@
+
+During this session, we will lead a collaborative review of the content so far and show Open Science Fair participants how they can contribute their knowledge to make it even better going forwards. This demo relates to the overall theme of the conference, as the *Turing Way* provides the tools to improve research habits in a self-contained handbook. It will also ensure that PhD students, postdocs, PIs and funding teams know which parts of the "responsibility of reproducibility" they can affect, and what they should do to nudge data science to being more efficient, effective and understandable.
+
+## Conference Themes
+
+*Select the conference theme(s) your proposal best addresses (remove the others):*
+
+Infrastructures for Open Science: services, methods, networks
+*	Open Access Platforms for all research artifacts
+*	Next Generation of Repositories
+*	Infrastructures for responsible metrics
+
+European Open Science Cloud (EOSC)
+*	FAIR data policy and practice: from theory to implementation
+*	EOSC in national settings
+*	Thematic Clouds 
+
+RRI and Open Science: bridging the gap
+*	Citizen Science and Public and Societal Engagement
+*	 Governance settings for Institutional embedded RRI and OS
+
+__Training and skills for Open Science__
+*	__Sustaining Open Science training: people and resources__
+*	FAIR competences for Higher Education
+
+Policies, Evaluation and Legal issues
+*	PlanS - principles, guidelines and implementation services
+*	Responsible metrics and research assessment
+*	GDPR and IPR exploitation 
+*	Rules of Participation in EOSC
+
+Innovative publishing and research dissemination
+*	Alternative publishing models 
+*	Open Peer Review
+*	Innovation on science communication
+
+Value added data products/services from Open Science 
+*	Research analytics and visualizations
+*	Text and data mining for/from research
+
+## Keywords
+
+*List 3-4 key terms that describe the subject of the proposal.*
+
+Reproducibile research, data science, open science, open research.
+
+## Audience
+
+*Identify the target audience. Some examples might be: Policy makers and funders, researchers, research Infrastructures and research communities, repository managers, publishers and content providers, libraries, research administrators, service providers and innovators.*
+
+Policy makers and funders, researchers, research Infrastructures and research communities, repository managers, publishers and content providers, libraries, research administrators.
```

Comment:
Weird capitalisation on "Infrastructures"?

Maybe emphasise the different types of researchers (ECRs, research software engineers and senior investigators?)